### PR TITLE
Issue #158 Remove two running info in o/p of start command

### LINKED ISSUE #158: Output of `crc start` contains duplicate info
When executing `crc start -b <bundle>`, it is reported twice that the CRC is running (at the end):
```
crc - Local OpenShift 4.x cluster
INFO  Checking if Virtualization is enabled       
INFO  Checking if KVM is enabled                  
INFO  Checking if Libvirt is installed            
INFO  Checking if user is part of libvirt group   
INFO  Checking if Libvirt is enabled              
INFO  Checking if Libvirt daemon is running       
INFO  Checking if crc-driver-libvirt is installed 
INFO  Checking if Libvirt crc network is available 
INFO  Checking if Libvirt crc network is active   
INFO  Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists 
INFO  Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists 
INFO  Extracting the Bundle tarball ...           
INFO  Starting stopped VM ...                     
INFO  Waiting 3m0s for the openshift cluster to be started ... 
INFO  To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/jsliacan/.crc/cache/crc_libvirt_4.1.0-rc.5/kubeconfig' 
INFO  Access the OpenShift web-console here: https://console-openshift-console.apps-crc.testing 
INFO  Login to the console with user: kubeadmin, password: 7cdWj-FtzNP-7VSYE-pZk7H 
INFO Running                                      
INFO Running 
```

### PR DESCRIPTION
**Pull request by anjannath**
Remove extra whitespace preceeding every console msg

Fixes #158

### CODE REVIEW

File: pkg/crc/machine/machine.go [hunk 1]
```diff
@@ -267,6 +267,6 @@ func setMachineLogging(logs bool) error {
 }
 
 func unsetMachineLogging() {
-	logging.CloseLogFile()
+	logging.CloseLogfile()
```
Comment (gbraad) [inline pkg/crc/machine/machine.go]: why not `CloseLogFile`?

**Comment by anjannath** [comment]
> Note: I guess it has to do with the logfile ?

Yes and No, to stop logrus from using the file to log to we have to remove the hooks, only closing the file is not sufficient. That's the fix to the double printing.

File: pkg/crc/machine/machine.go (hunk 1 above)
Comment (anjannath) [reply pkg/crc/machine/machine.go]: because there's a `OpenLogfile()`

File: pkg/crc/logging/logging.go [hunk 2]
```diff
@@ -75,10 +76,15 @@ func setupFileHook() error {
...
 }
 
-func CloseLogFile() {
+func CloseLogfile() {
 	logfile.Close()
 }
 
+func CloseLogging() {
+	CloseLogfile()
+	logrus.StandardLogger().ReplaceHooks(originalHooks)
```
Comment (anjannath) [inline pkg/crc/logging/logging.go]: Here is the fix for the double printing.

File: pkg/crc/logging/logging.go [hunk 3]
```diff
@@ -87,6 +93,10 @@ func InitLogrus(logLevel string) {
 		level = logrus.InfoLevel
 	}
 
+	for k, v := range logrus.StandardLogger().Hooks {
+		originalHooks[k] = v
+	}
```
Comment (anjannath) [inline pkg/crc/logging/logging.go]: and here we are storing the original hooks to later replace...

**Comment by gbraad** [comment]
I dislike seeing Closefile and CloseFile, and OpenFile and Openfile.
Inconsistent. If this is because of existing names, make them more distinct.

File: pkg/crc/machine/machine.go (hunk 1 above)
Comment (gbraad) [reply pkg/crc/machine/machine.go]: `LogFile` makes more sense... a File for Logging... a LogFile... right? It consist of two words: https://en.wikipedia.org/wiki/Log_file and not a single word. The convention is to use Camel Casing...

File: pkg/crc/machine/machine.go [hunk 4]
```diff
@@ -267,6 +267,6 @@ func setMachineLogging(logs bool) error {
...
 	logging.InitLogrus(logging.LogLevel)
```
Comment (gbraad) [inline pkg/crc/machine/machine.go]: just like, we call it a `LogLevel`, and not `Loglevel`

**Comment by anjannath** [comment]
@gbraad Updated!

**Comment by praveenkumar** [comment]
@anjannath every CI failed now, you need to check why.

**Comment by anjannath** [comment]
@praveenkumar It was an unused import that caused the CI failures, all green now!

File: pkg/crc/logging/logging.go [hunk 5]
```diff
@@ -75,10 +76,27 @@ func setupFileHook() error {
...
+func SetupFileHook() {
+	err := setupFileHook()
+	if err != nil {
+		logrus.Fatal(errors.Wrap(err, "Failed to open logfile"))
+	}
+}
+
+func RemoveFileHook() {
+	CloseLogFile()
+	logrus.StandardLogger().ReplaceHooks(originalHooks)
```
Comment (praveenkumar) [inline pkg/crc/logging/logging.go]: 👍

File: pkg/crc/machine/machine.go [hunk 6]
```diff
@@ -174,19 +174,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
...
 
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
-		logging.InfoF(" To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", result.ClusterConfig.KubeConfig)
-		logging.InfoF(" Access the OpenShift web-console here: %s", result.ClusterConfig.ClusterAPI)
-		logging.InfoF(" Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
+		logging.InfoF("To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", result.ClusterConfig.KubeConfig)
+		logging.InfoF("Access the OpenShift web-console here: %s", result.ClusterConfig.ClusterAPI)
+		logging.InfoF("Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
```
Comment (praveenkumar) [inline pkg/crc/machine/machine.go]: 👍

**Comment by praveenkumar** [comment]
/hold

@anjannath can you look this into that shouldn't be case.

```
16:30:24 INFO Adding user to libvirt group                 
16:30:24 panic: runtime error: invalid memory address or nil pointer dereference
16:30:24 [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xb0b415]
16:30:24 
16:30:24 goroutine 1 [running]:
16:30:24 github.com/code-ready/crc/pkg/crc/preflight.preflightCheckAndFix(0xce3500, 0xd04818, 0xd04878, 0xcde139, 0x1c, 0xc000253c00)
16:30:24 	/root/payload/pkg/crc/preflight/preflight_common.go:45 +0xf5
16:30:24 github.com/code-ready/crc/pkg/crc/preflight.SetupHost()
16:30:24 	/root/payload/pkg/crc/preflight/preflight_linux.go:98 +0x26c
16:30:24 github.com/code-ready/crc/cmd/crc/cmd.runSetup(0x1382f70, 0x0, 0x0)
16:30:24 	/root/payload/cmd/crc/cmd/setup.go:22 +0x20
16:30:24 github.com/code-ready/crc/cmd/crc/cmd.glob..func5(0x1358860, 0x1382f70, 0x0, 0x0)
16:30:24 	/root/payload/cmd/crc/cmd/setup.go:17 +0x3f
16:30:24 github.com/spf13/cobra.(*Command).execute(0x1358860, 0x1382f70, 0x0, 0x0, 0x1358860, 0x1382f70)
16:30:24 	/root/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
16:30:24 github.com/spf13/cobra.(*Command).ExecuteC(0x1358600, 0x14, 0x0, 0x0)
16:30:24 	/root/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
16:30:24 github.com/spf13/cobra.(*Command).Execute(0x1358600, 0xc000253f68, 0xb0e8e2)
16:30:24 	/root/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
16:30:24 github.com/code-ready/crc/cmd/crc/cmd.Execute()
16:30:24 	/root/payload/cmd/crc/cmd/root.go:64 +0x2d
16:30:24 main.main()
16:30:24 	/root/payload/cmd/crc/main.go:10 +0x25
```

**Comment by anjannath** [comment]
@praveenkumar This is the issue which we are fixing in #110 now since we have debug logging in the setup command, this error is coming from https://github.com/code-ready/crc/blob/master/pkg/crc/preflight/preflight_common.go#L45

**Comment by praveenkumar** [comment]
@anjannath you need to rebase it again.

**Comment by anjannath** [comment]
@praveenkumar Rebased!

**Pull request merged**